packages/cli/src/commands/ui/create/react.ts

```diff
@@ -141,12 +150,34 @@ export default class React extends Command {
     return true;
   }
 
-  private runReactCliCommand(args: string[], options = {}) {
-    return spawnProcess(
-      appendCmdIfWindows`npx`,
-      ['create-react-app'].concat(args),
-      options
-    );
+  private async runReactCliCommand(commandArgs: string[], options = {}) {
+    return new Promise<string>((resolve) => {
+      const child = spawnProcessPTY(
+        appendCmdIfWindows`npx`,
+        ['create-react-app'].concat([...commandArgs, '--color=always']),
+        options
+      );
+
+      const args = this.args;
+      let stopWritingInTerminal = false;
+      let remainingString = '';
```

Comment:
I think that if you create a buffer and write in it,  it might works without `--color=always` and be more efficient :)

Reply:
oups. The `--color-always` was added as an initial tentative to preserve the colors. Now that we use node-pty, I don't need it.